src/planner/operator/logical_hash_join.cpp

```diff
@@ -38,7 +38,8 @@ void LogicalHashJoin::computeFactorizedSchema() {
     switch (joinType) {
     case JoinType::INNER:
     case JoinType::LEFT:
-    case JoinType::COUNT: {
+    case JoinType::COUNT:
```

Comment:
Let's sort these switch case in alphabet order, it's getting long. Apply across this PR.